lib/kasket.rb

```diff
@@ -33,19 +33,24 @@ def setup(options = {})
     ActiveRecord::Base.extend(Kasket::ConfigurationMixin)
 
     if defined?(ActiveRecord::Relation)
-      ActiveRecord::Relation.send(:include, Kasket::RelationMixin)
-      Arel::SelectManager.send(:include, Kasket::SelectManagerMixin)
+      ActiveRecord::Relation.include Kasket::RelationMixin
+      Arel::SelectManager.include Kasket::SelectManagerMixin
     end
   end
 
   def self.cache_store=(options)
     @cache_store = ActiveSupport::Cache.lookup_store(options)
   end
 
-  def self.cache
+  def self.cache_store
```

Comment:
Why the rename of this method?

Reply:
Rubocop isn't happy that the memoize variable and the method name aren't the same...